joplin/templates/joplin_UI/primary_navigation.html

```diff
@@ -0,0 +1,31 @@
+{% load staticfiles wagtailadmin_tags %}
+{% load i18n %}
+
+{% block content %}
+
+<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css?family=Medium+IBM+Plex+Sans&display=swap" rel="stylesheet">
+
+<aside id="coa-primary_navigation" class="nav-wrapper" style="display: none;" data-nav-primary>
+  <div class="inner">
+    <a href="{% url 'wagtailadmin_home' %}" class="logo" aria-label="{% trans 'Dashboard' %}">
```

Comment:
Here's an example where i needed to add a template, but only had to add an Id to the top element to do all my styles. I just needed to write my SCSS to target the specific decedents of the element.